srcpkgs/helio-sequencer/template

```diff
@@ -0,0 +1,48 @@
+# Template file for 'helio-sequencer'
+pkgname=helio-sequencer
+version=3.12
+revision=1
+_gitrev="61f7db380f083b424418d65f4e26addc1370ddea"
+archs="i686 x86_64"
```

Comment:
This probably shouldn't be restricted to these architectures. Try building with all of our architectures, and fix whatever breaks unless upstream explicitly precludes building for those architectures.

Reply:
Ok. That sounds resonable. I just thought that i can provide x86 packages and then work on other archs. Cause cross-building is hard for me to understand and it didn't "just work"